### ISSUE TITLE
animationProgress is not updated during an animation

### ISSUE THREAD

**Issue by giosad** [airbnb/lottie-ios#369]
Hi, I want to sample the progress while the animation is playing. I need this so I can update some non-animatable stuff in sync with the animation.
 but it's not increasing until the animation finishes completely.
When I try to sample and print the animation progress in the `CADisplayLink` callback I added I get the following output:

```
16:45:22.774 progress 0
...
16:45:22.908 progress 0
16:45:22.924 progress 0
16:45:22.941 progress 0
16:45:22.958 progress 0
16:45:22.974 progress 1
```

Thanks a lot.

**Comment by buba447** [comment]
This is fixed in 2.0.5! Please Update and verify!

**Comment by giosad** [comment]
Validated, it works. Thank you.
